applications/GeoMechanicsApplication/tests/test_parameter_field.py

```diff
@@ -71,21 +73,29 @@ def test_parameter_field_with_python_umat_parameters(self):
         try:
             shutil.copy(custom_python_file, new_custom_script_path)
         except shutil.SameFileError as e:
-             print(f"Source and destination represents the same file. {e}")
-             raise
+            print(f"Source and destination represents the same file. {e}")
+            raise
```

Comment:
```suggestion
            raise RuntimeError("Source and destination represents the same file") from e
```

This way you get a proper stacktrace (`RuntimeError` can ofc be replaced by sth more appropriate

Reply:
Thank you for your suggestions. RuntimeError is used now.